.github/workflows/github-actions.yml

```diff
@@ -102,6 +120,8 @@ jobs:
             BUILD_TESTS: ON
             CMAKE_BUILD_TYPE: Release
             ENABLE_FORTRAN: OFF
+            ENABLE_MPI: OFF
+            ENABLE_OPENMP: OFF
```

Comment:
Should these (MPI/OPENMP) be enabled for the Release version as well? Thoughts?

Reply:
This is one of the test configs for CI, we do test Variorum's default build (RelWithDebugInfo) with MPI and OpenMPI on, as well as other variants of this too.